pdr_backend/cli/cli_module_lake.py

```diff
@@ -63,7 +64,7 @@ def do_lake_query(args, ppss):
 @enforce_types
 def do_lake_raw_drop(args, ppss):
     pds = PersistentDataStore(ppss.lake_ss.lake_dir, read_only=False)
-    drop_tables_from_st(pds, "raw", args.ST)
+    drop_tables_from_st(pds, "raw", UnixTimeMs(args.ST))
```

Comment:
this is not needed. The arg parser does the conversion and these are parsed args.

https://github.com/oceanprotocol/pdr-backend/blob/35b3c948e2fda776ab8ae21a6e02c35f6071707f/pdr_backend/cli/cli_arguments_lake.py#L23

Reply:
n.b. if args.ST were not a UnixTimeMs already, you would have to do UnixTimeMs.from_timestr(). Since args.ST is already an instance of UnixTimeMs and the constructor of UnixTimeMs takes in an int, what this change does is to reconvert the UnixTimeMs to an int and then back into a UnixTimeMs.